radixdlt/src/main/java/com/radixdlt/consensus/VertexStore.java

```diff
@@ -47,8 +48,7 @@
 	// Should never be null
 	private Vertex root;
 
-	// Should never be null
-	private QuorumCertificate highestQC;
+	private AtomicReference<QuorumCertificate> highestQC = new AtomicReference<>();
```

Comment:
What is the thinking here behind using AtomicReference?  The read/update in addQC still has a race condition, so not sure really what the win is over making this volatile?